github-runner-manager/src/github_runner_manager/cli.py

```diff
@@ -0,0 +1,30 @@
+# Copyright 2025 Canonical Ltd.
+# See LICENSE file for licensing details.
+
+"""The CLI entrypoint for github-runner-manager application."""
+
+from typing import TextIO
+
+import click
+
+from github_runner_manager.cli_config import Configuration
+
+
+# The entry point for the CLI will be tested with integration test.
+@click.command()
+@click.option(
+    "--config-file",
+    type=click.File(mode="r", encoding="utf-8"),
+    help="The file path containing the configurations.",
+)
+def main(config_file: TextIO) -> None:  # pragma: no cover
+    """Start the reconcile service.
+
+    Args:
+        config_file: The configuration file.
+
+    Raises:
+        NotImplementedError: The github runner manager logic is not yet implemented.
+    """
+    _ = Configuration.from_yaml_file(config_file)
+    raise NotImplementedError()
```

Comment:
This is missing this block of code at the end of the file:
```
if __name__ == "__main__":
    main()
```

I fixed this in WIP PR.

Reply:
Can I ask where the WIP PR is, and what is the approach to merging it here? I thought we were trying to merge things quickly into main.

Reply:
I have back port it to this PR.